src/server/cluster/outgoing_slot_migration.cc

```diff
@@ -20,7 +20,10 @@
 #include "server/server_family.h"
 #include "util/fibers/synchronization.h"
 
-ABSL_FLAG(int, slot_migration_connection_timeout_ms, 5000, "Timeout for network operations");
+ABSL_FLAG(int, slot_migration_connection_timeout_ms, 2000,
+          "Connection creating timeout for migration operations");
+ABSL_FLAG(int, migration_finalization_timeout_ms, 30000,
```

Comment:
we need to be able to config this flag via config set 
you need to use config_registry.RegisterMutable